tests/perf-test/OptionsSelector.perf-test.tsx

```diff
@@ -51,7 +57,7 @@ const mutlipleSectionsConfig = [
     {numItems: 1000, indexOffset: 0},
     {numItems: 100, indexOffset: 70},
 ];
-
+// @ts-expect-error TODO: Remove this once OptionsSelector is migrated to TypeScript.
```

Comment:
do we have any issue to track this @cead22 ? If not, should we be creating one?

Reply:
We have this one but it's closed https://github.com/Expensify/App/issues/25125. Maybe we need a new one for src/components/OptionsSelector/index.android.js and src/components/OptionsSelector/index.js